data-source/jsonnet/northern-ireland/blocks/individual/employment/employment_type.jsonnet

```diff
@@ -58,4 +58,18 @@ local proxyTitle = {
       when: [rules.proxyYes],
     },
   ],
+  routing_rules: [
+    {
+      goto: {
```

Comment:
When you define at least 1 routing rule, you must define a default rule. Ie currently you have `if`, but need the `else`. The default rule is just a routing rule without a `when` property.